src/lib/features/event-generator/components/atoms/StepButtons.svelte

```diff
@@ -18,16 +29,28 @@
 			</Button>
 		{/if}
 	</div>
-	<Button
-		on:click={() => activeStepStore.increment()}
-		size="large"
-		state={stepDataValid ? 'active' : 'disabled'}
-	>
-		{#if $stepsStore[$activeStepStore].button}
-			{$stepsStore[$activeStepStore].button.text}
-		{:else}
-			Next
-		{/if}
-		<Icon icon={$stepsStore[$activeStepStore].button.icon ?? 'tabler:arrow-right'} />
-	</Button>
+
+	{#if $stepsStore[$activeStepStore].button.text !== 'Create Event'}
+		<Button
+			on:click={() => activeStepStore.increment()}
+			size="large"
+			state={stepDataValid ? 'active' : 'disabled'}
+		>
+			{#if $stepsStore[$activeStepStore].button}
+				{$stepsStore[$activeStepStore].button.text}
+			{:else}
+				Next
+			{/if}
+			<Icon icon={$stepsStore[$activeStepStore].button.icon ?? 'tabler:arrow-right'} />
+		</Button>
+	{:else}
+		<Button on:click={captureFloat} size="large" state={stepDataValid ? 'active' : 'disabled'}>
```

Comment:
@IgnacioDebat little error here!

StepButtons component shouldn't be changed.
The action called by each step is not called from the StepButtons component. 
It is automatically called when the step changes, the action should be placed in the EventGeneratorSteps (I'll put a comment in the exact place).